src/modules/apropiaciones/dto/productos.dto.ts

```diff
@@ -0,0 +1,13 @@
+export class GeneralProducto {
+    vigencia: number;
+    nombre: string;
+    descripcion: string;
+    fechaCreacion: Date;
+    fechaModificacion: Date;
+    activo: boolean;
+}
+
+export class ProductoDTO {
+    general: GeneralProducto;
+    codigo: string;
+}
```

Comment:
(En general) Dejar vacía (sin espacios ni nada) la última línea de archivos de texto plano
```suggestion
}

```